Sources/ArcGISToolkit/Extensions/ArcGIS/SceneViewProxy.swift

```diff
@@ -17,6 +17,7 @@ import ArcGIS
 import ARKit
 import SwiftUI
 
+#if !os(visionOS)
```

Comment:
Could the enclosing declarations be annotated as unavailable on visionOS instead?